src/main/java/lotto/domain/Lotto.java

```diff
@@ -0,0 +1,49 @@
+package lotto.domain;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    private static final int START_RANGE = 1;
+    private static final int END_RANGE = 45;
+    private static final int COUNT = 6;
+    private final List<Integer> lottoNumbers;
+
+    public Lotto() {
+        this.lottoNumbers = generateRandomNumbers();
+    }
+
+    public Lotto(List<Integer> lottoNumbers) {
+        this.lottoNumbers = lottoNumbers;
+    }
+
+    private List<Integer> generateRandomNumbers() {
```

Comment:
아래와 같이 `생성자 메서드 체이닝`하도록 해주세요! 🤗 
```suggestion
        this(generateRandomNumbers());
    }

    public Lotto(List<Integer> lottoNumbers) {
        this.lottoNumbers = lottoNumbers;
    }

    private static List<Integer> generateRandomNumbers() {
```